orttraining/orttraining/python/training/ortmodule/_utils.py

```diff
@@ -97,3 +97,20 @@ def _create_iobinding(io_binding, inputs, model, device):
 
     for value_info in model.graph.output:
         io_binding.bind_output(value_info.name, device.type, device_id=get_device_index(device))
+
+class ModuleAccessor():
```

Comment:
I think we can name it in a more intuitive way. Accessor reminds users about getters and setters in OO

Maybe `ModelInfo` or `ModelMetadata` or something that gives a hint we use those internal models as a source of information for the original models